### PR TITLE
Initial work on a totally static hoc2015 learn page

### PR DESCRIPTION
**Pull request by breville**
The page is a super simple flat list of all the tutorials we want to list for a visiting user if we have switched our site to completely static.  I've just used image & title so far since we're not switching for language.

No external CSS at this point, just a single HTML file and images.  It's generated by viewing on a running pegasus instance, and the resulting page can be easily saved using chrome's "save page as.. | complete" and then tidying up the resulting file a little to simplify paths to the images.  (The page needs a title added to its head before saving so that the filenames are all kept simple.. until we get a title added properly in the page.)

Lots more styling can be done, but I want to get this reviewed for general direction.

@wjordan which of our own tutorials will we list?  what are their URLs?

@tanyaparker have I got the tutorial content correct?

It now uses JS, once the page loads, to show either MC+SWB+SWJS or SWB+SWJS+MC with a 50% split, then makes those items visible.

Here's a 50% scale image which shows the current state:

staticlearnpage2

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/statichoc2015/index.haml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  h3 {
+    margin-bottom: 0px;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+- tutorials = Tutorials.new(:tutorials)
+
+%a{href: "/mcstatic"}
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/statichoc2015/index.haml]: We still need to show one of these banners first, alternate between them, and show the other banner first in the list below.  This time, though, it'll have to be done in client-side Javascript (and without jquery, to minimize dependencies).

**Comment by alicesteinglass** [comment]
Generally looks good. If we're at a point where our whole site is down, my guess is the tutorials won't work either. So, I would only list the ones where we have a static fallback (Star Wars, Minecraft and Angry Birds). 

Don't put in links to any of our other tutorials - just go ahead and list the 3rd party ones after that.

File: pegasus/sites.v3/code.org/public/statichoc2015/index.haml (hunk 1 above)
Comment (breville) [inline pegasus/sites.v3/code.org/public/statichoc2015/index.haml]: I am not doing banners for now, but am randomly changing the order of the first two tutorials.

**Comment by caleybrock** [comment]
looking good :+1:

**Comment by tanyaparker** [comment]
Why does JS tutorial show up multiple times in the i18n section? Is that just the preview/not final screen shot?

**Comment by breville** [comment]
@tanyaparker good question .. those tutorials needed some extra differentiation which I've now added.  Screenshot updated.  (You might want to double check the languages.)

**Comment by tanyaparker** [comment]
OK to close this since we no longer need it?

**Comment by breville** [comment]
Now closed and branch deleted, but I see that github still offers a "Restore branch" option for this PR which is nice, in case we need it like next hoc :)

**Comment by philbogle** [comment]
I believe it would be prudent to have a nicely designed static version on tap for HOC2016.   It doesn't have to be done this instant, but as a forcing function I would advocate having this ready before I leave at the end of the June.

**Pull request closed without merging**